tooling/Microsoft.VisualStudio.BlazorExtension/Properties/AssemblyInfo.cs

```diff
@@ -8,13 +8,6 @@
 //
 // The versions here need to match what the build is producing. If you change the version numbers
 // for the Blazor assemblies, this needs to change as well.
-[assembly: ProvideBindingRedirection(
-    AssemblyName = "Microsoft.AspNetCore.Blazor.AngleSharp",
-    GenerateCodeBase = true,
-    PublicKeyToken = "",
-    OldVersionLowerBound = "0.0.0.0",
-    OldVersionUpperBound = "0.9.9.0",
-    NewVersion = "0.9.9.0")]
```

Comment:
This should still be here, but it should refer to the updated assmblyname and version

Reply:
Do you know what i need to write here.

I am always getting this error in VS
![image](https://user-images.githubusercontent.com/873751/40553701-c251a78e-6043-11e8-8f8a-0506b38493a3.png)

Reply:
My guess is that it would be 0.9.9.2 - it should be the assembly version.

Reply:
Seems that it should be 0.9.9.0 
![image](https://user-images.githubusercontent.com/873751/40555562-d61ee078-6049-11e8-90f9-c57869a3a6e2.png)

Reply:
Ok